src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IInvocationOperation.vb

```diff
@@ -0,0 +1,45 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub IInvocation_SharedMethodWithInstanceReceiver()
```

Comment:
>IInvocation_SharedMethodWithInstanceReceiver [](start = 19, length = 44)

Please make sure we have tests for accessing a method (static and instance) of off a type

Reply:
Done.

---
In reply to: [149249472](https://github.com/dotnet/roslyn/pull/22972#discussion_r149249472) [](ancestors = 149249472)